exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -42,266 +41,316 @@ git submodule add https://github.com/526avijitgupta/gokarna.git themes/gokarna
+### e. Launching the Website Locally {#launching-the-website-locally}
 
-Launch by using the following command:
+Use the [`server` subcommand](https://gohugo.io/commands/hugo_server/#synopsis) to view your site.
 
 ```bash
-hugo serve
+hugo server
 ```
 
-Go to `http://localhost:1313`.
+Go to [`http://localhost:1313`](http://localhost:1313/) (if no other Hugo servers are running; Hugo will use an [ephemeral port](https://en.wikipedia.org/wiki/Ephemeral_port) if one or more servers are running concurrently on the host).
 
-### f. Build the Website
+### f. Build the Website {#build-the-website}
 
-When your site is ready to deploy, run the following command:
+Use the `hugo` command to build your site.
 
 ```bash
 hugo
 ```
 
-A `public` folder will be generated, containing all static content and assets for your website. It can now be deployed on any web server.
+A `public` folder will be generated, containing all static content and assets for your website. It can be hosted on any web server.
 
-The website can be automatically published and hosted with [Netlify](https://www.netlify.com/), [AWS Amplify](https://gohugo.io/hosting-and-deployment/hosting-on-aws-amplify/), [Github pages](https://gohugo.io/hosting-and-deployment/hosting-on-github/), [Render](https://gohugo.io/hosting-and-deployment/hosting-on-render/) and more...
+The website can be automatically published and hosted with [Netlify](https://www.netlify.com/), [AWS Amplify](https://gohugo.io/hosting-and-deployment/hosting-on-aws-amplify/), [GitHub Pages](https://gohugo.io/hosting-and-deployment/hosting-on-github/), [Render](https://gohugo.io/hosting-and-deployment/hosting-on-render/), and more...
 
 ## Configuration
 
-In addition to [Hugo global configuration](https://gohugo.io/overview/configuration/) and [menu configuration](#basic-configuration), **gokarna** lets you define the following parameters in your site configuration (here is a `config.toml`, whose values are default).
+In addition to [Hugo global configuration](https://gohugo.io/overview/configuration/) and [menu configuration](#basic-configuration), Gokarna lets you define the following [parameters](https://gohugo.io/methods/site/params/) in your site configuration.
+
+See this sample `config.toml`, which uses Gokarna's default values:
 
 ```toml
 [params]
-  # URL for the avatar on homepage
+  # Resource URL for the avatar (seen on the home page and header)
   avatarURL = ""
 
-  # Choose one of size-xs, size-s, size-m, size-l & size-xl. (Default: size-m)
-  avatarSize = ""
+  # Describe the avatar for screen readers
+  avatarAltText = ""
 
-  # Description to display on homepage
-  description = "Sky above, sand below & peace within"
+  # Set the avatar's size. Valid values include:
+  # size-xs, size-s, size-m, size-l & size-xl
+  avatarSize = "size-m"
 
-  # Accent color is displayed when you hover over <a> tags
+  # Choose the color shown when hyperlinks are hovered over
   accentColor = "#FF4D4D"
```

Comment:
nit: `accentColor` should be the first in alphabetical order, before any of the avatar* properties